lib/audio_net.py

```diff
@@ -94,23 +100,20 @@ def __init__(self, dataset, net_count = 1, audio_settings = None):
         
         split = int(np.floor(self.validation_split * self.dataset_size))
 
+        # Split the dataset into validation and training data loaders
+        dataset_rng = torch.Generator().manual_seed(self.data_seed)
+        train_dataset, val_dataset = torch.utils.data.random_split(dataset, [self.dataset_size - split, split], generator=dataset_rng)
```

Comment:
Note that dataset now has its own seed, and we do the split once, rather than doing it once per ensemble member